src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Return Ball with number:" + number + " and " + color + " colore";

```

Comment:
There is a typo in the `toString` method. The word 'colore' should be corrected to 'color'.